src/sidebar/util/test/type-coercions-test.js

```diff
@@ -0,0 +1,139 @@
+import { toBoolean, toInteger, toObject, toString } from '../type-coercions';
+
+describe('sidebar/type-coercions', () => {
+  describe('#toBoolean', () => {
```

Comment:
```suggestion
  describe('toBoolean', () => {
```

The `#` prefix comes from JSDoc and indicates an instance method or property. If you're just referring to a non-method function, it should be omitted.